mysql-test/main/func_misc.result

```diff
@@ -1641,3 +1641,89 @@ DROP TABLE t1;
+SELECT RELEASE_ALL_LOCKS();
+RELEASE_ALL_LOCKS()
+0
+# Test function with one lock only
+SELECT GET_LOCK('l1',10);
+GET_LOCK('l1',10)
+1
+SELECT RELEASE_ALL_LOCKS();
+RELEASE_ALL_LOCKS()
+1
+SELECT LOCK_MODE, LOCK_TYPE, TABLE_SCHEMA, TABLE_NAME FROM information_schema.metadata_lock_info WHERE thread_id>0 ORDER BY TABLE_NAME;
+LOCK_MODE	LOCK_TYPE	TABLE_SCHEMA	TABLE_NAME
+# Test function with multiple locks
+SELECT GET_LOCK('l1',10),
+GET_LOCK('l2',10),
+GET_LOCK('l3',10),
+GET_LOCK('l4',10),
+GET_LOCK('l5',10),
+GET_LOCK('l6',10),
+GET_LOCK('l7',10),
+GET_LOCK('l8',10),
+GET_LOCK('l9',10),
+GET_LOCK('l10',10),
+GET_LOCK('l11',10),
+GET_LOCK('l12',10),
+GET_LOCK('l13',10),
+GET_LOCK('l14',10),
+GET_LOCK('l15',10);
+GET_LOCK('l1',10)	GET_LOCK('l2',10)	GET_LOCK('l3',10)	GET_LOCK('l4',10)	GET_LOCK('l5',10)	GET_LOCK('l6',10)	GET_LOCK('l7',10)	GET_LOCK('l8',10)	GET_LOCK('l9',10)	GET_LOCK('l10',10)	GET_LOCK('l11',10)	GET_LOCK('l12',10)	GET_LOCK('l13',10)	GET_LOCK('l14',10)	GET_LOCK('l15',10)
+1	1	1	1	1	1	1	1	1	1	1	1	1	1	1
+SELECT LOCK_MODE, LOCK_TYPE, TABLE_SCHEMA, TABLE_NAME FROM information_schema.metadata_lock_info WHERE thread_id>0 ORDER BY TABLE_NAME;
+LOCK_MODE	LOCK_TYPE	TABLE_SCHEMA	TABLE_NAME
+MDL_SHARED_NO_WRITE	User lock	l1	
+MDL_SHARED_NO_WRITE	User lock	l7	
+MDL_SHARED_NO_WRITE	User lock	l6	
+MDL_SHARED_NO_WRITE	User lock	l13	
+MDL_SHARED_NO_WRITE	User lock	l9	
+MDL_SHARED_NO_WRITE	User lock	l3	
+MDL_SHARED_NO_WRITE	User lock	l14	
+MDL_SHARED_NO_WRITE	User lock	l11	
+MDL_SHARED_NO_WRITE	User lock	l10	
+MDL_SHARED_NO_WRITE	User lock	l8	
+MDL_SHARED_NO_WRITE	User lock	l2	
+MDL_SHARED_NO_WRITE	User lock	l15	
+MDL_SHARED_NO_WRITE	User lock	l5	
+MDL_SHARED_NO_WRITE	User lock	l4	
+MDL_SHARED_NO_WRITE	User lock	l12	
+SELECT RELEASE_ALL_LOCKS();
+RELEASE_ALL_LOCKS()
+15
+SELECT LOCK_MODE, LOCK_TYPE, TABLE_SCHEMA, TABLE_NAME FROM information_schema.metadata_lock_info WHERE thread_id>0 ORDER BY TABLE_NAME;
+LOCK_MODE	LOCK_TYPE	TABLE_SCHEMA	TABLE_NAME
+# Test function with recursive locks
+SELECT GET_LOCK('l1',10),
+GET_LOCK('l2',10),
+GET_LOCK('l2',10),
+GET_LOCK('l3',10),
+GET_LOCK('l3',10),
+GET_LOCK('l3',10),
+GET_LOCK('l4',10),
+GET_LOCK('l4',10),
+GET_LOCK('l4',10),
+GET_LOCK('l4',10),
+GET_LOCK('l5',10),
+GET_LOCK('l5',10),
+GET_LOCK('l5',10),
+GET_LOCK('l5',10),
+GET_LOCK('l5',10);
+GET_LOCK('l1',10)	GET_LOCK('l2',10)	GET_LOCK('l2',10)	GET_LOCK('l3',10)	GET_LOCK('l3',10)	GET_LOCK('l3',10)	GET_LOCK('l4',10)	GET_LOCK('l4',10)	GET_LOCK('l4',10)	GET_LOCK('l4',10)	GET_LOCK('l5',10)	GET_LOCK('l5',10)	GET_LOCK('l5',10)	GET_LOCK('l5',10)	GET_LOCK('l5',10)
+1	1	1	1	1	1	1	1	1	1	1	1	1	1	1
+SELECT LOCK_MODE, LOCK_TYPE, TABLE_SCHEMA, TABLE_NAME FROM information_schema.metadata_lock_info WHERE thread_id>0 ORDER BY TABLE_NAME;
+LOCK_MODE	LOCK_TYPE	TABLE_SCHEMA	TABLE_NAME
+MDL_SHARED_NO_WRITE	User lock	l4	
+MDL_SHARED_NO_WRITE	User lock	l1	
+MDL_SHARED_NO_WRITE	User lock	l2	
+MDL_SHARED_NO_WRITE	User lock	l5	
+MDL_SHARED_NO_WRITE	User lock	l3	
```

Comment:
Unfortunately `ORDER BY TABLE_NAME` didn't affect ordering, it still looks odd. Apparently lock name is encoded into `TABLE_SCHEMA`. If so, please change it to `ORDER BY TABLE_SCHEMA`. And omit `TABLE_NAME` from the query, it doesn't make sense here.

Reply:
Corrected at https://github.com/Daniel-Solo/server/commit/f0ca3bbc10a877dcbac3515e912fbf070cdc02dc